Source/Scene/CesiumIon.js

```diff
@@ -91,7 +91,34 @@ define([
      *   });
      */
     CesiumIon.createResource = function(assetId, options) {
+        var endpointResource = CesiumIon.createEndpointResource(assetId, options);
+
+        return CesiumIon._loadJson(endpointResource)
+            .then(function (endpoint) {
+
+                var externalType = endpoint.externalType;
+                if (!defined(externalType)) {
+                    return CesiumIonResource.create(endpoint, endpointResource);
+                }
+
+                // 3D Tiles and STK Terrain Server external assets can still be represented as a resource
+                // object, just not the CesiumIonResource object.
+                if (externalType === '3DTILES' || externalType === 'STK_TERRAIN_SERVER') {
+                    return new Resource({ url: endpoint.options.url });
+                }
+
+                //External imagery assets have additional configuration that can't be represented as a Resource
+                throw new RuntimeError('CesiumIon.createResource does not support external imagery assets.');
```

Comment:
Add a comment to use `CesiumIon.createImageryProvider` instead.  Or can't we just make a call to that from this function instead of throwing an error?

Reply:
I agree on updating the error message, but I think just calling it from here would be confusing since it's a completely different return type.  I also expect external assets to change a lot in the future, so they'll remain experimental for longer.

Reply:
👍